aiida/cmdline/commands/cmd_export.py

```diff
@@ -80,7 +80,7 @@ def inspect(archive, version, data, meta_data):
 @options.GROUPS()
 @options.NODES()
 @options.ARCHIVE_FORMAT(
-    type=click.Choice(['zip', 'zip-uncompressed', 'tar.gz', 'null']),
+    type=click.Choice(['zip', 'zip-uncompressed', 'zip-lowmemory', 'tar.gz', 'null']),
```

Comment:
Do we want to expose this at the CLI level already?

We just need to keep in mind that this means we should continue supporting this option until 2.0.
(probably ok)

Reply:
yep I'm happy with supporting it